applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,1783 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Riccardo Tosi
+//
+
+#include "symbolic_d_convection_diffusion_explicit.h"
+
+namespace Kratos
+{
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::SymbolicDConvectionDiffusionExplicit(
+    IndexType NewId,
+    GeometryType::Pointer pGeometry)
+    : SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>(NewId, pGeometry) {}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::SymbolicDConvectionDiffusionExplicit(
+    IndexType NewId,
+    GeometryType::Pointer pGeometry,
+    Properties::Pointer pProperties)
+    : SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>(NewId, pGeometry, pProperties) {}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::~SymbolicDConvectionDiffusionExplicit() {}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    NodesArrayType const& ThisNodes,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, this->GetGeometry().Create(ThisNodes), pProperties);
+}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    GeometryType::Pointer pGeom,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, pGeom, pProperties);
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateLocalSystem(
```

Comment:
Shouldn't this method be exactly the same as in the QS base class?

Reply:
Also note that in the explicit case this method shouldn't be needed.

Reply:
I removed the method. Concerning the first comment, I couldn't derive it from the QS class. I have als spoken with @roigcarlo about this in July, and we could not find a solution.
However, since the method is removed, I close the isssue.